misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/README.md

```diff
@@ -1,23 +1,30 @@
 ## Current data mapping
+
 ****
-Values computed in Metasfresh 
+Values computed in Metasfresh
```

Comment:
- Is this a section header? If yes, please mark it as one.
- I don't understand this line in context with the following line.
  `Order` - is it computed in metasfresh or pulled via the search endpoint?

Reply:
PS: :nerd_face: it's metasfresh with a small "m"

Reply:
Regarding product/price-mappings: AFAIS https://github.com/metasfresh/metasfresh/pull/11805 has been only pushed to master. How should I proceed?